File: src/main/java/guess/gameFor.java
package guess;

import java.util.Random;
import java.util.Scanner;

public class gameFor {
    public static void main(String[] args) {
        Random random = new Random();
        Scanner scanner = new Scanner(System.in);
        int secret = random.nextInt(10)+1;
        System.out.print("secret="+secret);

        int num = 0;
        for (int i = 0 ; i < 5 ; i++ ){
            System.out.print("Please enter a number("+i+"/4)：");
            num = scanner.nextInt();
            if(num > secret)
                System.out.println("Smaller");
            else if (num < secret)
                System.out.println("Bigger");
            else{
                System.out.println("great!!");
                break;
            }
        }
    }
}

File: src/main/kotlin/com/GameFor.kt
package com

import java.util.*
import kotlin.random.Random

fun main(args: Array<String>) {

    var secret = Random.nextInt(1,10)
    println("secret=$secret")
    val scanner = Scanner(System.`in`)
    for(i in 1..4){
        println("Please enter a number(${i}/4)：")
        var number = scanner.nextInt()
        println("第${i}次 $number")
        if(secret > number)
            println("Bigger")
        else if (secret < number)
            println("Smaller")
        else if (secret == number) {
            println("great!! the num is $secret")
            break
        }
        if (number == -1){
            break
        }
    }
}